Formalisation(shacl)/Core/PiecesShape/Dataset.ttl

```diff
@@ -1,177 +1,391 @@
+    "Key elements that represent an individual in the dataset."@en;
+  sh:name "personal data"@en;
+  sh:nodeKind sh:IRI;
+  sh:path dqv:hasPersonalData .
+
+<http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#population-coverage> sh:datatype
+    xsd:string;
+  sh:description "A definition of the population within the dataset"@en;
+  sh:name "population coverage"@en;
+  sh:nodeKind sh:Literal;
+  sh:path healthdcatap:populationCoverage .
+
+<http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#publisher> sh:description
+    "An entity (organisation) responsible for making the Dataset available."@en;
+  sh:maxCount 1;
+  sh:minCount 1;
+  sh:name "publisher"@en;
+  sh:node foaf:Agent;
+  sh:path dct:publisher .
+
+<http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#publisher-note> sh:datatype
+    xsd:string;
+  sh:description "A description of the publisher activities"@en;
+  sh:maxCount 1;
+  sh:name "publisher note"@en;
+  sh:nodeKind sh:Literal;
+  sh:path healthdcatap:publishernote .
+
+<http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#publisher-type> sh:description
+    "A type of organisation that makes the Dataset available"@en;
+  sh:maxCount 1;
+  sh:name "publisher type"@en;
+  sh:nodeKind sh:IRI;
+  sh:path healthdcatap:publishertype .
+
+<http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#purpose> sh:description
+    "A free text statement of the purpose of the processing of data or personal data."@en;
+  sh:name "purpose"@en;
+  sh:nodeKind sh:IRI;
+  sh:path dqv:hasPurpose .
+
+<http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#qualified-attribution>
+  sh:description "An Agent having some form of responsibility for the resource"@en;
+  sh:name "qualified attribution"@en;
+  sh:nodeKind sh:IRI;
+  sh:path prov:qualifiedAttribution .
+
+<http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#qualified-relation> sh:description
+    "A description of a relationship with another resource"@en;
+  sh:name "qualified relation"@en;
+  sh:nodeKind sh:IRI;
+  sh:path dcat:qualifiedRelation .
+
+<http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#quality-annotation> sh:description
+    "A statement related to quality of the Dataset, including rating, quality certificate, feedback that can be associated to the dataset"@en;
+  sh:name "quality annotation"@en;
+  sh:nodeKind sh:IRI;
+  sh:path dqv:hasQualityAnnotation .
+
+<http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#release-date> sh:datatype
+    xsd:dateTime;
+  sh:description "The date of formal issuance (e.g., publication) of the Dataset."@en;
+  sh:maxCount 1;
+  sh:name "release date"@en;
+  sh:nodeKind sh:Literal;
+  sh:path dct:issued .
+
+<http://data.health-ri.nl/core/p2/DatasetShapeDatasetShape#retention-period> sh:description
+    "A temporal period which the dataset is available for secondary use"@en;
+  sh:name "retention period"@en;
+  sh:nodeKind sh:IRI;
+  sh:path healthdcatap:retentionperiod .
```

Comment:
See the issue raised in https://github.com/Health-RI/health-ri-metadata/issues/166.

```suggestion
  sh:path healthdcatap:retentionPeriod .
```